app/bundles/CoreBundle/IpLookup/AbstractMaxmindLookup.php

```diff
@@ -11,10 +11,7 @@
 
 namespace Mautic\CoreBundle\IpLookup;
 
-/**
- * Class MaxmindLookup.
- */
-class MaxmindLookup extends AbstractRemoteDataLookup
+abstract class AbstractMaxmindLookup extends AbstractRemoteDataLookup
```

Comment:
Don't forget to report these BC breaks into https://docs.google.com/document/d/1lHNK8hHQtvyWlz9OtfAoLupDVb5-QunQjq-HAGlAsS4/edit#heading=h.qork18mo6yws

Reply:
Done